src/course-team/course-team-member/CourseTeamMember.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Badge, Button } from '@edx/paragon';
+import { DeleteOutline } from '@edx/paragon/icons';
+import messages from './messages';
+import { USER_ROLES } from '../enum';
+
+const CourseTeamMember = ({
+  userName,
+  role,
+  email,
+  onChangeRole,
+  onDelete,
+  currentUserEmail,
+  isHideActions,
+  isAllowActions,
+}) => {
+  const intl = useIntl();
+  const isCurrentUser = currentUserEmail === email;
+
+  const badgeText = `${role === USER_ROLES.admin
+    ? intl.formatMessage(messages.roleAdmin)
+    : intl.formatMessage(messages.roleStaff)}`;
+
+  return (
+    <div className="course-team-member" data-testid="course-team-member">
+      <div className="member-info">
+        <Badge variant={role === USER_ROLES.admin ? 'danger' : 'secondary'}>
+          {badgeText}
+          {isCurrentUser ? (
+            <span className="badge-currentUser">{intl.formatMessage(messages.roleYou)}</span>
+          ) : null}
+        </Badge>
+        <span className="member-info-name">{userName}</span>
+        <a className="member-info-email" href={`mailto:${email}`}>{email}</a>
+      </div>
+      {/* eslint-disable-next-line no-nested-ternary */}
+      {isAllowActions ? (
```

Comment:
Maybe better and more readable

```suggestion
      {isAllowActions && (
```